app/src/views/private/components/sidebar-detail.vue

```diff
@@ -59,6 +59,11 @@ export default defineComponent({
 		const { sidebarOpen } = toRefs(appStore);
 		return { active, toggle, sidebarOpen };
 	},
+	methods: {
+		closeSidebar() {
+			useAppStore().setSidebarOpen(false);
+		},
+	},
```

Comment:
We shouldn't mix Vue Composition API and Options API.

Reply:
Good point! Anyway, the method has been removed as it's not needed anymore :)